### ISSUE TITLE
KCD Nigeria 2025

### ISSUE THREAD

**Issue by thisisobate** [cncf/kubernetes-community-days#500]
# KCD Organizer Checklist
Please fill in information, organizers, and sponsors as you assemble them, and then make a new comment when this application is ready for review.

### Event Details
* [ ] Event Location: Lagos, Nigeria
* [x] Event Date and Time: TBD
* [x] Expected Number of Attendees: 500
* [x] Event cost per ticket: Free
#### Event type:
* [ ] Virtual
* [x] Hybrid
* [ ] In person
#### Organizers Details (identify at least 3 organizers). Designate who will be the CFP admin, as well as the finanical officer:
* [x] Organizer 1: Uchechukwu Obasi, CNCF, Developer Advocate, obasiuche62@gmail.com, @uche
* [x] Organizer 2: Ileriayo Adebiyi, Container Solutions, Cloud Nate Engineer, ileriayoadebiyi@gmail.com
* [x] Organizer 3: Edidiong Asikpo, Ambassador Labs, Senior Developer Advocate, edidiongasikpo@gmail.com
* [x] Organizer 4: Somtochi Onyekwere, Weaveworks, Developer Experience Engineer, somtochionyekwere@gmail.com

### Terms

* [x] No other Kubernetes Community Day event in same major city (check with past organizers)
* [x] Minimum of 3 organizers from 3 different organizations
* [ ] Every organizer has an account on [Cloud Native Community Groups](https://community.cncf.io/)
* [ ] Each organizer lives in or just outside of the KCD location
* [ ] At least one organizer is a CNCF member employee, CNCF ambassador, or CNCF project maintainer
* [ ] All organizers agree to follow the event guidelines and CNCF Code of Conduct
* [ ] All inquiries / communication to come via GitHub (comment under your KCD issue) for easy tracking
* [ ] All organizers have taken [Inclusive Open Source Community Orientation (LFC102)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
* [ ] Agree to offer at least 3 complimentary tickets to increase diversity for the event
* [ ] Share the sponsor prospectus with CNCF via your GitHub issue, and link it on your website.
* [ ] Agree to ensure the final program schedule is diverse (e.g., not all speakers of one gender, one culture, or from 1 or 2 of the same company). [Here are some tips on how to create a diverse lineup.](https://docs.google.com/presentation/d/1fzT_BdavVKh3mnxxU-PBWyJq9JUfasKwHqekkbYVbw8/edit#slide=id.g56245ab439_0_106)
* [ ] Agree to submit your final lineup to kcd@cncf.io for review, prior to publishing it
* [ ] Agree to use Cloud Native Community Groups for registration, scheduling, and event website. If using a platform outside of Bevy, organizers must provide CNCF with a full list of attendees.
-- Regardless of your platform choice, please use [these questions](https://docs.google.com/spreadsheets/d/1OEoVQ8Y3eQmEEJeT2abhNOkYIkyk8nAVjebNFvKMTQQ/edit?usp=sharing) to create your registration form
* [ ] Agree to use SM Apply for CFP and to close it at least 6 weeks before the event

### Getting started checklist for organizers

* [ ] Schedule monthly check-ins with KCD Core Organizer(s) (kcd@cncf.io)
* [ ] Minimum of 3 sponsors committed (not required for virtual only events)
Sponsor 1
Sponsor 2
Sponsor 3
* [ ] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
Fiscal sponsor that will accept payments
* [ ] Added usernames for [CNCF Slack](https://slack.cncf.io/) for all organizers so that we can add you to the #kcd-organizers channel
* [ ] Create a Twitter handle specific to your KCD event (except for countries like China and Korea where Twitter is not used)

### Getting started checklist for CNCF:
* [x] Organizer email alias has been created: [organizers-nigeria@kubernetescommunitydays.org](mailto:organizers-nigeria@kubernetescommunitydays.org)
* [x] Region created in Bevy
* [ ] Added to Bevy header
* [ ] CFP created in SMapply
* [ ] Marketing Plan shared

### Organizer Planning Timeline
* [ ] Make a copy of [this timeline](https://docs.google.com/spreadsheets/d/1bvCiyyDut1seSnBE6pzVevcJkXLeWbxbncvhFsyY8PI/edit#gid=1736864170) based on whether your event is In-person, Hybrid, or Virtual and link your own spreadsheet here ().
* [ ] To assure you have a diverse lineup, you are required to send a copy of your program to kcd@cncf.io before you publish it.

### Post event checklist for Organizer:
* [ ] Send CNCF event analytics
* [ ] Schedule post mortem meeting
* [ ] Send CNCF videos to be uploaded to YouTube
* [ ] Link videos from YouTube to event page
* [ ] Post event survey sent out

### Post event checklist for CNCF:
* [ ] Removed from Bevy header
* [ ] CFP deactivated
* [ ] Upload videos to YouTube

**Comment by antonio-pedro99** [comment]
@thisisobate Hi. Why this event is called KCD Africa?

**Comment by antonio-pedro99** [comment]
@thisisobate I understood this, but why Africa and not specifically Nigeria or wherever the event is going to happens?
I am saying this because we have KCD Angola(#486), which is also in Africa.

**Comment by AudMonte01** [comment]
Yes, @thisisobate - due to there already being another KCD in Africa in the planning phases, we will need to name this one by city just like Angola.

Also, September 22-23 seems quite soon because:
* We recommend the full schedule be released at least 6 weeks ahead of the event
* Two weeks prior to the agenda release, is reviewing and then time for speakers to affirm their acceptance
* Ahead of that the call for proposals (CFP) should be open for at least 1 month

All of this cannot begin until a venue is secured, which is really what will determine your dates. Let's have a call to discuss this further: https://calendly.com/audmonte/kcd-onboarding-kickoff-call

**Comment by thisisobate** [comment]
Yes...I agree with both of you @antonio-pedro99 @AudMonte01 
I think it's time for us to name it by city just like Angola.

Happy to discuss further about our prep plans @AudMonte01

**Comment by AudMonte01** [comment]
JULY 20 KICK-OFF MEETING

History
* 2022 was going to be hybrid and went virtual last-minute due to lack of sponsorships
* 2021 had over 650 registrations and about 50% check-in rate
* Used Open Collective platform account for fiscal sponsorship

Budget
* Rough budget of $4,000 USD for the entire event
-- $475 USD for one of the venue
* Audra to sent Uche sponsorship template in Canva

Goal
* NOTE: Lagos is a tech hub in Africa
* 2-day event w/ 1st day virtual and and 2nd day is in-person
* Virtual goal would be 350
* In-person attendee goal will be goal of 150 (looking to stream)

Action Items
* [ ] Uche looking into Open Collective account
* [ ] @k8tgreenley please create an email alias for these organizers
* [ ] Audra to create organizer private slack channel

Feedback
* Eventbrite is used in Nigeria, but tickets will be free this year

**Comment by k8tgreenley** [comment]
Here is the email alias: organizers-nigeria@kubernetescommunitydays.org

**Comment by AudMonte01** [comment]
AUGUST 4 MEETING NOTES

New event dates
* Google Dev Fest is either Nov or Dec (Edi to reach out to organizers)

Potential Venue
* [Zone Tech Park](https://www.instagram.com/zonetechpark/?hl=en)
* Uche looking into Open Collective account, but seems active based on monthly emails

Sponsorships
* Uche is helping with sponsorship deck
* Will engage with Grafana labs marketing team
* GitLab is next on the list

**Comment by AudMonte01** [comment]
OCTOBER 6 MEETING NOTES

Having a hard time with sponsorships
* Most have already spent their budget for the year
* Ileri and Uche are okay with postponing to next year in the 2nd quarter
* [ ] Ileri and Uche will touch base with Somto and Edi for opinions on a new date

**Comment by AudMonte01** [comment]
NOVEMBER 29 MEETING UPDATES

Venue
* Co-location with OSCAFEST? Edi is a co-organizer
-- Event will occur dependent on sponsorships. Likely to happen, but not 100%
-- Conversation with that organizing team will happen in December/January

Community Activity
* Ileriayo working in the Lagos chapter hosting meetups

**Comment by AudMonte01** [comment]
CNCF SYNC on FEBRUARY 27

Team assignments will come when OSCAFEST comes together to start planning

Logo 
* Audra shared examples from other KCDs
* Audra shared raw files with team to build out their logo
* [ ] Team to either put together their logo or reach out to kcd@cncf.io to leverage the LF Creative team

Sponsorships
* Still would like to approach RedHat
* Isovalent would be great sponsor
* [ ] Uche to have OSCAFEST organizer reach out to CNCF events team with their sponsor prospectus to inquire about sponsoring the 3rd-party event

**Comment by AudMonte01** [comment]
Thank you for your submission - we are now ready to accept official applications via Survey Monkey. The deadline for submissions will be Wednesday September 11 at 11:59pm Pacific Daylight Time. More information and the application can be found via https://www.cncf.io/kcds/application/.

This GitHub issue is now closed. Should your KCD be accepted into the 2025 program, a core CNCF staff member will open a new issue for program tracking and transparency.
